static/js/publisher/release/actions/pendingReleases.test.js

```diff
@@ -225,4 +227,42 @@ describe("pendingReleases actions", () => {
       expect(resumeProgressiveRelease(key).payload).toBe(key);
     });
   });
+
+  describe("cancelProgressiverelease", () => {
+    const key = "progressive-test";
+    const previousRevision = {
+      architectures: ["amd64"],
+      attributes: {},
+      base: "core18",
+      build_url: null,
+      channels: ["latest/edge"],
+      confinement: "strict",
+      created_at: "2019-07-16T08:58:04Z",
+      epoch: { read: null, write: null },
+      grade: "stable",
+      revision: 3,
+      "sha3-384": "test",
+      size: 4096,
+      status: "Published",
+      version: "1.8.0"
```

Comment:
Wow, providing just an object with id or any necessary fields would be enough, but sure, we can be very specific ;)